### PR TITLE
Install rust toolchain

### PR DESCRIPTION
**Pull request by jmesnil**
before building & testing the extractor

### CODE REVIEW

File: Containerfile.rust-dev [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN chown dev:dev /home/dev
 # From here we run everything as dev user
 USER       dev
 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.81.0 -y
```
Comment (tremes) [inline Containerfile.rust-dev:14]: I don't know Rust versioning, but would something like `latest` make sense?

Comment (jmesnil) [reply Containerfile.rust-dev:14]: I prefer to stick to a version like we do for Go https://github.com/openshift/insights-operator/blob/5e2dd6f1e127c4108a70e94d13debf333a751b7a/go.mod#L5

By using `latest` we would have unreproduceable builds and updates that we might break the code.

Comment (tremes) [reply Containerfile.rust-dev:14]: OK OK :)

**Comment by tremes** [comment]
It seems to work fine for me! 
/lgtm

**Pull request merged**